test/powershell/engine/Basic/ValidateAttributes.Tests.ps1

```diff
@@ -316,10 +313,13 @@ Describe 'Validate Attributes Tests' -Tags 'CI' {
             $byteArray = [System.IO.File]::ReadAllBytes($filePath)
             $byteList  = [System.Collections.Generic.List[byte]] $byteArray
             $byteCollection = [System.Collections.ObjectModel.Collection[byte]] $byteArray
-            ## Use the running time of 'MandatoryFunc -Value $byteArray' as the baseline time
-            $baseline = (Measure-Command { MandatoryFunc -Value $byteArray }).Milliseconds
-            ## Running time should be less than 'expected'
-            $expected = $baseline + 20
+            ## Running time should be less than 'UpperBoundTime'
+            ## This is not really a performance test (perf test cannot run reliably in our CI), but a test
+            ## to make sure we don't check the elements of a value-type collection.
+            ## The crossgen'ed 'S.M.A.dll' is about 28mb in size, and it would take over 2000ms if we check
+            ## each byte of the array, list or collection. We use 200ms as the upper bound value in tests to
+            ## prove that we don't check each byte.
+            $UpperBoundTime = 200
```

Comment:
Previous implementation used some MandatoryFunc to establish a reasonable baseline with adding some extra time to that.
Can $UpperBoundTime use similar runtime-defined baseline (and adding something on top of that) rather than a constant?

Reply:
I guess I can add back the baseline and add 200 ms to it